sway/commands.c

```diff
@@ -408,6 +417,63 @@ static struct cmd_results *cmd_border(int argc, char **argv) {
 	return cmd_results_new(CMD_SUCCESS, NULL, NULL);
 }
 
+static struct cmd_results *parse_border_color(struct border_colors *border_colors, const char *cmd_name, int argc, char **argv) {
+	if (argc != 5) {
+		return cmd_results_new(CMD_INVALID, cmd_name, "Requires exact 5 color values");
+	}
+
+	uint32_t colors[5];
+	int i;
+	for (i = 0; i < 5; i++) {
+		if (sscanf(argv[i], "#%x", &colors[i]) != 1) {
+			return cmd_results_new(CMD_INVALID, cmd_name, "Invalid color value");
+		}
+	}
+
+	// Colors are in RGB format, add alpha channel
```

Comment:
It should be possible to optionally define the alpha channel in the config (even though i3 doesn't support that).